app/views/layouts/main.hbs

```diff
@@ -41,7 +41,7 @@
     <nav id="nav" class='navbar text-center'>
       <div class="row">
         {{#if logged_in}}
-          <span class="glyphicon glyphicon-menu-hamburger" style="cursor:pointer" onclick="openNav()"></span>
+        <span><input class="hamburger" type="image" style="cursor:pointer" onclick="openNav()" src="/images/Hamburger NAV.svg" /></span>
```

Comment:
Should probably remove the space in the filename. I suppose it will work as is with most or all webservers, but it just seems a bit funky and to me not worth the risk that some server/CDN/etc might barf on the space.

Also, out of curiosity, why use an `<input>` tag vs just an `<img>` tag?